pkg/controllers/monitoring-stack/controller.go

```diff
@@ -196,3 +220,35 @@ func (r *reconciler) reconcileObject(ctx context.Context, ms *stack.MonitoringSt
 	logger.Info("Updating stack component")
 	return r.k8sClient.Update(ctx, desired)
 }
+
+func (r *reconciler) createGrafanaDSWatch(ctx context.Context) error {
+	if r.grafanaDSWatchCreated {
+		return nil
+	}
+	log := r.logger.WithName("create-grafana-ds-watch")
+	var dataSources grafanav1alpha1.GrafanaDataSourceList
+
+	err := r.k8sClient.List(ctx, &dataSources, client.InNamespace("default"))
+
+	if err != nil {
```

Comment:
Failing to list shouldn't return a error since it isn't an actual reconcile error.